examples/autoinstrument/README.md

```diff
@@ -53,6 +53,40 @@ Or, if you'd like to synthesize a parent trace:
 curl -H "traceparent:  00-ff000000000000000000000000000041-ff00000000000041-01" ${cluster_ip}
 ```
 
+## Running in Google Cloud Run
+
+To run this example in Google Cloud Run, you need to run the convenience script provided. After following the prerequisites, 
+
+First, export the Google Cloud Region for cloud run to `GOOGLE_CLOUD_RUN_REGION` environment variable:
+
+```shell
+# This can be any supported Google cloud region
+export GOOGLE_CLOUD_RUN_REGION=us-central1
+```
+
+Then, from the root of the repository,
+```shell
+cd examples/autoinstrument && ./run_in_cloud-run.sh
+```
+This will deploy the containerized application to Cloud Run and you will be presented with a service URL which would look something like - 
+
+```text
+Service URL: https://hello-autoinstrument-cloud-run-m43qtxry5q-uc.a.run.app
+```
+
```

Comment:
Now that we're using the proxy, the service URL should no longer matter right?

Reply:
Yes, this is just to clarify what to expect in case of a successful deployment.